website/blog/2023-06-05-ferretdb-new-release-v-1-3-0.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: ferretdb-new-release-v-1-3-0
```

Comment:
It does not match the title

Reply:
For blog posts, slugs don't need to match the title because they are quite long most times. Instead generic terms are taken out leaving only the keywords.

Reply:
Great point. Updating it now